### PR TITLE
[terra-application-links] Add styling and fix spec tests.

### PR DESCRIPTION
**Pull request by dkasper-was-taken**
### Summary
Missing spec tests for the open menu slipped past me, this corrects the tests and the styling.

@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-application-links/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```
Comment (emilyrohrbough) [inline packages/terra-application-links/CHANGELOG.md]: Should this changelog heading be Fixed?

File: packages/terra-application-links/tests/wdio/application-tabs-spec.js [hunk 2]
```diff
@@ -1,118 +1,128 @@
 const viewports = Terra.viewports('small', 'large');
 
 describe('ApplicationTabs', () => {
-  describe('Displays a default application tabs', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-default');
-      browser.waitForVisible('#test-tabs [data-application-tabs-more]');
-      browser.moveToObject('#test-tabs [data-application-tabs-more]');
+  viewports.forEach((viewport) => {
```
Comment (emilyrohrbough) [inline packages/terra-application-links/tests/wdio/application-tabs-spec.js]: You might need this loop outside the describe. I am seeing this screenshot different for the large viewport size

screen shot 2019-02-15 at 12 09 05 pm

Comment (dkasper-was-taken) [reply packages/terra-application-links/tests/wdio/application-tabs-spec.js]: Updated good catch, that fixed it.

File: packages/terra-application-links/CHANGELOG.md (hunk 1 above)
Comment (dkasper-was-taken) [reply packages/terra-application-links/CHANGELOG.md]: Updated.

**Pull request merged**